packages/notifications/src/index.ts

```diff
@@ -1 +1,3 @@
+export * from './error';
```

Comment:
Should `withErrorHandling` and `withErrorIgnoring` be exported? They're more internals for `NotificationProvider`. On their own, they're not really notifications related.

Ideally... these might go in `core-utils`?

Reply:
I think I'd vote not export them in this PR and then we can move them (and deal with the related version bump requirements) in a follow on?

Reply:
a smaller followup PR would be good... but they have to be exported since they are re-exported in src/libs/error.ts for pretty wide use across terra-ui codebase.

Reply:
Oh, I see.

Reply:
Alternatively, libs/error could temporarily duplicate these functions until a follow up adds them to core-utils where both notifications and Terra UI can pull them from.